merge/README.md

```diff
@@ -36,6 +36,14 @@ For most cases, this may not be the most efficient solution. See [the migration
     # Optional. Default is '*'
     pattern:
 
+    # The desired behavior if no artifacts are found using the provided pattern.
+    # Available Options:
+    #   warn: Output a warning but do not fail the action
+    #   error: Fail the action with an error message
+    #   ignore: Do not output any warnings or errors, the action does not fail
+    # Optional. Default is 'error'
+    if-no-files-found:
```

Comment:
AFAIK we're talking about artifacts, not files. That's why I would call it at least `if-no-artifacts-found`. However, some other wording options (I don't have a strong opinion here, just some thoughts to find a good solution):

- `no-artifacts-behavior` / options: `["warn", "error", "ignore"]`
- `no-artifacts-behavior` / options: `["exit-with-warning", "fail", "exit-gracefully"]`